packages/main/scripts/download-remote-extensions.spec.ts

```diff
@@ -28,6 +28,7 @@ import product from '/@product.json' with { type: 'json' };
 
 import type { RemoteExtension } from './download-remote-extensions.js';
 import { downloadExtension, getRemoteExtensions, main } from './download-remote-extensions.js';
+import ErrnoException = NodeJS.ErrnoException;
```

Comment:
question: why do we need to have such import vs only using NodeJS.ErrnoException where it's needed ? (line 80)